unified-installer/astra-unified-installer.sh

```diff
@@ -2488,31 +2592,51 @@ step_existing_trident_flags_compatibility_check() {
 }
 
 step_generate_trident_fresh_install_yaml() {
-    local -r kustomization_file="$__GENERATED_KUSTOMIZATION_FILE"
+    local -r trident_kustomization_file="$__GENERATED_TRIDENT_KUSTOMIZATION_FILE"
```

Comment:
Same as the connector; this function now only operators on the trident kustomization file.